models/containerized_model/templates/python/Dockerfile.ubi9

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023. Cognitive Scale Inc. All rights reserved.
+# Licensed under CognitiveScale Example Code License https://github.com/CognitiveScale/cortex-certifai-examples/blob/master/LICENSE.md
+#
+FROM {{BASE_DOCKER_IMAGE}}
+
+COPY requirements.txt /tmp/
+RUN pip install -r /tmp/requirements.txt
+
+COPY src /containerized_model/src
+COPY model /containerized_model/model
```

Comment:
FYI @mchutani-CS - The `src/` and `model/` folders are under the `containerize_model/` folder. Let me know, if you would prefer these to be at the root level

Reply:
@mchutani-CS  I've updated the Dockerfile's to have sufficient permissions for writing to `/src`, `/model` and `/license` directories. The final prediction service image will use these directories when storing the Prediction Service (py file) and the model.pkl at run time.